### ISSUE TITLE
Search packages feature in the UI

### ISSUE THREAD

**Issue by Baalekshan** [glasskube/glasskube#950]
**Is your feature request related to a problem? Please describe.**
Add a search functionality in the UI to search the packages easily.

**Describe the solution you'd like**
User can search the package in search bar with its name and it shows the installed packages as well as options to install new packages.
As well as there should label/filters that can be used to filter packages.

Suggested Filters/Labels could be
```Installed```
```Not Installed ```

Based on the scope: 
```Cluster```
```Namespace```
 In the namespace part, index the available namespaces to choose from.

**Describe alternatives you've considered**

**Additional context**

**Comment by babautkarsh** [comment]
Hi can i work on this enhancement

**Comment by pmig** [comment]
@babautkarsh This task does not have the help wanted label. Did you already work with htmx and go templates?

**Comment by babautkarsh** [comment]
yes i have experience in Go primarily ,TypeScript in Frontend Development and also with templating engines

**Comment by christophenne** [comment]
@babautkarsh this is quite a big feature and will require lots of discussion. For a first time contributor, we recommend picking up an easier issue first, e.g. like #1071.

**Comment by pmig** [comment]
A single search bar the search over all packages could either be placed on the nav bar. We should create to different search inputs for cluster and namespace scoped packages.
